spec/integration/inventory_spec.rb

```diff
@@ -99,6 +99,21 @@ module Hawkular::Inventory::RSpec
       expect(resources.size).to be(2)
     end
 
+    it 'Should list feeds when using SSL without certificate' do
+      # change this to the real credentials when updating the VCR
+      @state[:super_secret_username] = 'username'
+      @state[:super_secret_password] = 'password'
+      creds = { username: @state[:super_secret_username],
+                password: @state[:super_secret_password] }
+      tori_url = 'https://hawkular.torii.gva.redhat.com/hawkular/inventory'
```

Comment:
a unitest should not include references to such environments, especially if not everyone have access to them.